actions/admin/paymail_addresses.go

```diff
@@ -61,8 +79,20 @@ func (a *Action) paymailAddressesCount(w http.ResponseWriter, req *http.Request,
 }
 
 // paymailCreateAddress will create a new paymail address
+// Create Paymail godoc
+// @Summary      Create paymail
+// @Description  Create paymail
+// @Tags		 Admin
+// @Param key query string true "key"
+// @Param address query string true "address"
+// @Param public_name query string false "public_name"
+// @Param avatar query string false "avatar"
+// @Param metadata query string false "metadata"
+// @Produce      json
+// @Success      200
```

Comment:
Not your fault. But this is a "Create" method. But success code is 200(OK). Shouldn't it be 201(Created)?